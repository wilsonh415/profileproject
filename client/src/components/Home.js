import React from 'react';

class Home extends React.Component {
    render() {
        const styles = {
            position: "relative",
            top: "17%",
            fontFamily: "Georgia",
            color: "white"
        }
        return (
            <div className="homeStyle">
                <h1 style={styles}>
                    Welcome! Website is still a work in progress.
                </h1>
            </div>
        )
    }
}

export default Home;