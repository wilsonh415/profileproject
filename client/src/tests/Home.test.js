import React from 'react';
import { shallow } from 'enzyme';
import { configure } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import Home from '../components/Home';
import renderer from 'react-test-renderer';

configure({ adapter: new Adapter() });

test('Home component renders properly', () => {
    const component = renderer.create(<Home></Home>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

test('Title in home component is correct', () => {
    const component = shallow(<Home></Home>);
    const title = component.find("h1").text();
    expect(title).toEqual("Hi, I'm Wilson. Welcome to my website!");
});

