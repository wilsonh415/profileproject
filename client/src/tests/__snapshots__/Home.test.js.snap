// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<div
  className="homeStyle"
>
  <h1
    style={
      Object {
        "color": "white",
        "fontFamily": "Georgia",
        "fontSize": "45px",
        "marginLeft": "10vw",
        "marginRight": "10vw",
        "position": "relative",
        "top": "19%",
      }
    }
  >
    Hi, I'm Wilson. Welcome to my website!
  </h1>
</div>
`;
